.github/workflows/build.yml

```
@@ -26,7 +26,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
           SONAR_TOKEN: ${{secrets.SONAR_TOKEN}}
-        run: ./mvnw -B clean verify sonar:sonar -Dsonar.token=${{ secrets.SONAR_TOKEN }}
+        run: ./mvnw -B clean verify sonar:sonar -Dsonar.login=${{ secrets.SONAR_TOKEN }}
```

As SONAR_TOKEN is set in env, it is not required to add sonar.login

```suggestion
        run: ./mvnw -B clean verify sonar:sonar
```